mmdet/ops/nms/nms_wrapper.py

```diff
@@ -93,7 +96,7 @@ def nms(boxes, scores, iou_threshold, score_thr=0.0, max_num=-1, offset=0):
         inds = order.masked_select(select)
     else:
         if max_num < 0:
-            max_num = sys.maxsize
+            max_num = boxes.shape[0]
```

Comment:
BTW, will it appear in a graph as a constant? If yes, I still doubt if this is a valid way to resolve the issue. Is the problem with this on OpenVINO side or ONNX Runtime also handles this improperly?

Reply:
Ok, seems that I finally got the reason of it. 😄 Then, the question would be: is it done the same way in `ote` branch?

Reply:
No, it will not appear in a graph as a constant. I rewrote the symbolic, so that the graph will have the corresponding nodes to get the value `max_num`, depending on the shape of the input `boxes`.
<img width="607" alt="nms_nodes" src="https://user-images.githubusercontent.com/10177297/114141449-42920f80-991a-11eb-8da5-a287750fe47c.PNG">

No, in the ote branch the `max_num` value is defined as a constant:
`max_num = int(dets_th.size(0))`
Therefore, this value is fixed in a graph and cannot be changed when the model is reshaped.

Performance from ote branch (benchmark_app.py, sync, CPU):
Count:      94 iterations
Duration:   60426.72 ms
Latency:    616.36 ms
Throughput: 1.62 FPS

Reply:
Awesome! Thanks!